src/models/httpResponse.ts

```diff
@@ -18,6 +18,7 @@ export interface HttpResponse {
   request?: Request;
 
   timings?: HttpTimings;
+  durationMillis?: number;
```

Comment:
The Property seems out of place here. The duration does not show the duration of the response. But rather the duration for the whole processing of the httpRegion. 
My idea would be to move the creation of the data object (ProcessedHttpRegion) to a single interceptor and therefore avoid these properties.

Reply:
I see, so we want to move the logic of creating toProcessedHttpRegion out of logResponseInterceptor into a new processedHttpRegionInterceptor and have the beforeLoop set the start duration and the afterLoop take the completed httpResponse and generate the final ProcessedHttpRegion is that correct?

Reply:
yes, this was my idea. One small Problem. The logResponseInterceptor should create a copy of the response and call logResponse Hook. This copy is also needed in the new processedHttpRegionInterceptor. We need a nice way to geht this response.